README.md

```diff
@@ -583,22 +572,37 @@ In this case, you will need to configure the following additional settings that
 Nexus instance that hosts the permanent content: `cachito_nexus_hoster_username`,
 `cachito_nexus_hoster_password`, and `cachito_nexus_hoster_url`.
 
+On each request, Cachito will create a proxy repository to the group repository of the request's
+package manager (e.g. `cachito-pkg-manager`). Cachito will populate this proxy repository to
+contain the subset of dependencies declared in the repository's lock file. Once populated, Cachito
+will block the repository from getting additional content. This prevents the consumer of the
+repository from installing something that was not declared in the lock file. This is further
+enforced by locking down the repository to a single user created for the request, which the
+consumer will use. Please keep in mind that for this to function properly, anonymous access needs
+to be disabled on the Nexus instance or at least not set to have read access on all repositories.
+
+These repositories and users created per request are deleted when the request is marked as stale
+or the request fails.
+
 ## Package Managers
 
 ### Feature Support
 
 The table below shows the supported package managers and their support level in Cachito.
 
-Feature                 | gomod | npm | pip |
----                     | ---   | --- | --- |
-Baseline                | ✓     | ✓   | ✓   |
-Content Manifest        | ✓     | ✓   | ✓   |
-Dependency Replacements | ✓     | x   | x   |
-Dev Dependencies        | ✓     | ✓   | ✓   |
-External Dependencies   | N/A   | ✓   | ✓   |
-Multiple Paths          | ✓     | ✓   | ✓   |
-Nested Dependencies     | ✓     | ✓   | x   |
-Offline Installations   | ✓     | x   | x   |
+Feature                 |  gomod  | npm | pip   | yarn           |
+---                     |  ---    | --- | ---   | ---            |
+Versions supported      | >= 1.13 | < 7 | >= 10 | >= 1.12.0, < 2 |
+Baseline                |  ✓      | ✓   | ✓     | ✓              |
+Content Manifest        |  ✓      | ✓   | ✓     | ✓              |
+Dependency Replacements |  ✓      | x   | x     | x              |
+Dev Dependencies        |  ✓      | ✓   | ✓     | ✓[^1]          |
+External Dependencies   |  N/A    | ✓   | ✓     | ✓              |
+Multiple Paths          |  ✓      | ✓   | ✓     | ✓              |
+Nested Dependencies     |  ✓      | ✓   | x     | ✓              |
+Offline Installations   |  ✓      | x   | x     | x              |
+
+[^1] Cachito keeps Dev Dependencies for yarn packages but can not distinguish them as it done for npm.
```

Comment:
I don't think we need extra explanation for this. IMO this is what `x` means.

It *could* be misunderstood as "yarn does not download dev dependencies at all," but if we wanted this table to be precise, we would have more exceptions than rules.

Maybe the explanation for Dev Dependencies should clearly state that it only applies to Cachito's ability to determine if something is a dev dependency or not.

Reply:
I believe Athos's point about adding clarification was for gomod, which is reasonable but does not need to be part of this PR I think.